patterns/README.md

```diff
@@ -1030,3 +1030,363 @@ A field MAY have a `missingValues` property that MUST be an `array` where each e
+   schema targeted toward a specific piece of software). While the extensions
+   are intended to support the use of features not available in all software,
+   the resulting data package should continue to work as well as possible with
+   software that does not have those features.
+2. Related to (1), extensions should only include metadata that describe the
+   data themselves—not instructions for what a specific software package should
+   do with the data. Users who want to include the latter may do so within
+   a sub-namespace such as `custom` (e.g., see Issues [#103](https://github.com/frictionlessdata/specs/issues/103)
+   and [#663](https://github.com/frictionlessdata/specs/issues/663)).
+3. Extensions must be backward compatible (i.e., not break existing tools,
+   workflows, etc. for working with Frictionless packages).
+
+It is worth emphasizing that the scope of the proposed extensions is strictly
+limited to the information necessaary to make use of the features for working
+with categorical data provided by the software packages listed above. Previous
+discussions of this issue have occasionally included references to additional
+variable-level metadata (e.g., multiple sets of category labels such as both
+"short labels" and longer "descriptions", or links to common data elements,
+controlled vocabularies or rdfTypes). While these additional metadata are
+undoubtedly useful, we speculate that the large majority of users who would
+benefit from the extensions propopsed here would not have and/or utilize such
+information, and therefore argue that these should be considered under a
+separate proposal.
+
+### Implementations
+
+Our proposal to add a field-specific `enumOrdered` property has been raised
+[here](https://github.com/frictionlessdata/specs/issues/739) and
+[here](https://github.com/frictionlessdata/specs/issues/156).
+
+Discussions regarding supporting software providing features for working with
+categorical variables appear in the following GitHub issues:
+
+- [https://github.com/frictionlessdata/specs/issues/156](https://github.com/frictionlessdata/specs/issues/156)
+- [https://github.com/frictionlessdata/specs/issues/739](https://github.com/frictionlessdata/specs/issues/739)
+
+and in the Frictionless Data forum:
+
+- [https://discuss.okfn.org/t/can-you-add-code-descriptions-to-a-data-package/](https://discuss.okfn.org/t/can-you-add-code-descriptions-to-a-data-package/)
+- [https://discuss.okfn.org/t/something-like-rs-ordered-factors-or-enums-as-column-type/](https://discuss.okfn.org/t/something-like-rs-ordered-factors-or-enums-as-column-type/)
+
+Finally, while we are unaware of any existing implementations intended for
+general use, it is likely that many users are already exploiting the fact that
+arbitrary fields may be added to the
+[table schema](https://specs.frictionlessdata.io//table-schema/)
+to support internal implementations (e.g., our group is doing so).
```

Comment:
Maybe drop the "our group is doing so" as it is unclear to which group this refers?

Reply:
Good suggestion; done.